Dockerfiles/agent/windows/amd64/Dockerfile

```diff
@@ -6,11 +6,17 @@ ARG BASE_IMAGE=mcr.microsoft.com/windows/servercore:1809
 #ARG BASE_IMAGE=mcr.microsoft.com/powershell:nanoserver-1809
 FROM ${BASE_IMAGE} 
 
+ARG WITH_JMX="false"
+
 LABEL maintainer "Datadog <package@datadoghq.com>"
 
+USER ContainerAdministrator
+
 COPY datadog-agent-7-latest.amd64.zip install.ps1 ./
 RUN powershell ./install.ps1
 
+RUN if "%WITH_JMX%" == "false" (echo WITH_JMX not set, JRE not installed) else (java -version)

```

Comment:
Sounds more like a debug line? I think it's better to put it inside `install.ps1`

Reply:
👍